<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QWidget" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>10</y>
          <width>200</width>
          <height>30</height>
        </rect>
      </property>
      <property name="text">
        <string>Agregar Documento</string>
      </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>50</y>
          <width>200</width>
          <height>30</height>
        </rect>
      </property>
      <property name="text">
        <string>Eliminar Documento</string>
      </property>
    </widget>
    <widget class="QPushButton" name="btnList">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>90</y>
          <width>200</width>
          <height>30</height>
        </rect>
      </property>
      <property name="text">
        <string>Listar Documentos</string>
      </property>
    </widget>
    <widget class="QLineEdit" name="lineSearch">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>130</y>
          <width>200</width>
          <height>30</height>
        </rect>
      </property>
      <property name="placeholderText">
        <string>Buscar Documento...</string>
      </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
      <property name="geometry">
        <rect>
          <x>10</x>
          <y>170</y>
          <width>380</width>
          <height>100</height>
        </rect>
      </property>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>
